src/consensus/pbft/pbfttypes.h

```diff
@@ -24,4 +24,36 @@ namespace pbft
     NodeId from_node;
   };
 #pragma pack(pop)
+
```

Comment:
Since this is identical with the one in rafftypes.h (minus the deserialise) it would be better to move this into a separate file and have both raft and pbft use that instead of duplicating.

Reply:
I think it is better to have them separate so that PBFT has a limited interface to the store, and not access to deserialise().